src/Presentation/RangeFacetHtmlBuilder.php

```diff
@@ -22,13 +22,17 @@ public function __construct(
 	 * TODO: add smoke integration test
 	 */
 	public function buildHtml( FacetConfig $config, PropertyConstraints $state ): string {
+		$currentMin = $state->getInclusiveMinimum();
+		$currentMax = $state->getInclusiveMaximum();
+
 		return $this->parser->processTemplate(
 			'RangeFacet',
 			[
 				'msg-min' => wfMessage( 'wikibase-faceted-search-facet-range-min' )->text(),
 				'msg-max' => wfMessage( 'wikibase-faceted-search-facet-range-max' )->text(),
-				'current-min' => $state->getInclusiveMinimum(),
-				'current-max' => $state->getInclusiveMaximum(),
+				'msg-apply' => wfMessage( 'wikibase-faceted-search-facet-apply' )->text(),
+				'current-min' => $currentMin,
+				'current-max' => $currentMax
```

Comment:
Why did you introduce variables here? 

I think the old form is better/simpler, though it does not matter much as this is all local

Reply:
It was a leftover from [bcc1cfc](https://github.com/ProfessionalWiki/WikibaseFacetedSearch/pull/100/commits/bcc1cfcaa470b91491fbbea6aaa6987ba2d1baf8). I have created a follow-up PR: https://github.com/ProfessionalWiki/WikibaseFacetedSearch/pull/103